tests/PrepareBodyMiddlewareTest.php

```diff
@@ -73,6 +73,24 @@ function (RequestInterface $request) {
         $this->assertEquals(200, $response->getStatusCode());
     }
 
+    public function testAddsContentLengthForGET()
```

Comment:
I've reused the existing test now, and added tests for methods with and without bodies